docs/platform/server-v3/modules/common-services/data-sources.md

```diff
@@ -0,0 +1,280 @@
+---
+id: data-sources
+title: Data Sources in Falcon Server
+sidebar_label: Data Sources
+enterprise_only: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[Apollo Server documentation](https://www.apollographql.com/docs/apollo-server/data/data-sources/) says:
+
+> Data sources are classes that Apollo Server can use to encapsulate fetching data from a particular source, such as a database or a REST API. These classes help handle caching, deduplication, and errors while resolving operations.
+
+Falcon Server Data Sources extends [Apollo Server Data Sources](https://www.apollographql.com/docs/apollo-server/data/data-sources/) this is why if you are not familiar with DataSources concept we strongly recommend learning that topic first.
+
+In Falcon ecosystem Data Sources are used to fetch data from all the back-ends, e.g. if you have integration with BigCommerce then [BigCommerce Module](./modules/bigcommerce-module) exposes `FalconBigCommerceDataSource` which implements the communication layer between Falcon and BigCommerce. It contains authentication and authorization logic and does everything that's needed to fetch the required data.
+
+The same applies to all the integrations - there's always an integration-specific Data Source that realizes the communication.
+
+Falcon Server will treat particular service as Data Source until it extends [Apollo's abstract Data Source class](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-datasource/src/index.ts). However, the Falcon Server expects that concrete DataSource has an additional `name` property. This is why each DataSource registered in Falcon Server needs to:
```

Comment:
```suggestion
Falcon Server will treat one service as Data Source [Apollo's abstract Data Source class](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-datasource/src/index.ts), but expects that new DataSource has an additional `name` property. This is why each DataSource registered in Falcon Server needs to:
```

Reply:
I am sorry but I think current version has greater informative value, it is less ambiguous